app/src/main/AndroidManifest.xml

```diff
@@ -14,13 +14,16 @@
     <application
         android:name=".ui.App"
         android:allowBackup="true"
+        android:fullBackupContent="true"
```

Comment:
Why

Reply:
using fullBackupContent backs up the user data when a person uninstalls the application, he can restore the state after reinstalling the application

Reply:
How is it related to this issue?

Reply:
It was one of the lint fixes suggested @iamareebjamal.